figureMS3.svg

```diff
@@ -0,0 +1,4888 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Don't think you meant to commit this?

Reply:
deleted

Reply:
Do you mean running the statistical testing with k-means and GMM clusters as well?

Reply:
I don't think you need to make that comparison everywhere.